lib/api.js

```diff
@@ -27,7 +27,7 @@ function merge (a, b, mapper) {
       !(b[k] instanceof Uint8Array) &&
       !Array.isArray(b[k])
     ) {
-      a[k] = {}
+      a[k] ??= {}
```

Comment:
:melt: how did this previously work? It seems like it used to overwrite, but now you're changing that.
I just woke up, so not sure implications but I would put a :red flag: next to this to recheck

Reply:
I would argue it does nothing because it recurses into merge until it goes into that else branch then overwrites anyway...

Reply:
Yes it used to overwrite `a[k]`. I'm shocked that it even worked before in modules downstream. Now it only assigns `{}` to `a[k]` if `a[k]` is not there.